src/TaskPlanner/Contracts/Requests/PerformSwitchoverRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Middleware.TaskPlanner.Contracts.Requests;
+
+public record PerformSwitchoverRequest
+{
+    public Guid ActionPlanId { get; init; }
+    public Guid InstanceId { get; init; }
```

Comment:
We can only switch over one instance at a time? We cannot have a List<InstanceId> or a Dicctionary of InstanceId and DestionationName

Reply:
Moving more than one instance would be beneficial for us. But I'm afraid I have to disagree with passing a list of deployed instance identifiers. I want to suggest the middle ground. We will pass the id of action, allowing us to move all the instances deployed within a single action. What do you think?

Reply:
Changed